lib/Elastica/Exception/ResponseException.php

```diff
@@ -31,7 +31,7 @@ public function __construct(Request $request, Response $response)
     {
         $this->_request = $request;
         $this->_response = $response;
-        parent::__construct($response->getError());
+        parent::__construct(json_encode($response->getError()));
```

Comment:
This is temporary workaround that fixes fatal error (now getError return array instead of string).